pkg/logger/log/log.go

```diff
@@ -18,9 +18,14 @@ import (
 	"github.com/emicklei/go-restful/v3"
 )
 
-const MaskedQueryParams = "MaskedQueryParams"
-const MaskedRequestFields = "MaskedRequestFields"
-const MaskedResponseFields = "MaskedResponseFields"
+const (
+	MaskedQueryParamsAttribute    = "MaskedQueryParams"
+	MaskedRequestFieldsAttribute  = "MaskedRequestFields"
+	MaskedResponseFieldsAttribute = "MaskedResponseFields"
+	UserIDAttribute               = "Log-UserId"
```

Comment:
is it possible to use same format for these attribute.eg: Log-UserId => LogUserId and  Log-ClientId => LogClientId

Reply:
sure